_pages/clients/r.md

```diff
@@ -56,207 +54,219 @@ If you are running [Algorithmia Enterprise](/enterprise), you can specify the AP
+{: .notice-info}
 
-Your account can make up to {{site.data.stats.platform.max_num_algo_requests}} Algorithmia requests at the same time (this limit <a onclick="Intercom('show')">can be raised</a> if needed).
+### Download the resulting file
 
-## Conclusion
+We can download the resulting file using the URI returned in the algorithm output. First, verify that the file that you want to download exists, and download it to a new local file:
 
-This guide covered installing Algorithmia via R, uploading and downloading data to and from a user created data collection, checking if a file exists using the Data API, calling an algorithm, and handling errors.
+{% highlight r %}
+# Download the file
+if (client$file(response$result).exists()) {
+    local_file <- client$file(response$result)$getFile()
+}
+{% endhighlight %}
 
-For more information on the methods available using the Data API in R check out the [Data API](http://docs.algorithmia.com/?r#data-api-specification) documentation or the [R Client Docs](https://github.com/algorithmiaio/algorithmia-r).
+This copies the file from your data collection and saves it as a file on your local machine, storing the filename in the variable `local_file`.
 
-For convenience, here is the whole script available to run:
+Alternately, if you just need the JSON content of the processed file to be stored in a variable, you can retrieve the remote file's content without saving the actual file:
 
 {% highlight r %}
-library(algorithmia)
+# Download contents of file as a string
+if client$file(response$result)$exists()) {
+    processed_text <- client$file(download_uri).getJson()
+}
+    
+{% endhighlight %}
 
-# Authenticate with your API key
-apiKey <- "YOUR_API_KEY"
-# Create the Algorithmia Client object
-client <- getAlgorithmiaClient(apiKey)
+This will get the file as a JSON object, saving it to the variable `processed_text`, which might be useful when writing algorithms that are part of a text processing pipeline.
 
-# Instantiate a DataDirectory object, set your data URI and call Create
-nlp_directory <- client$dir("data://YOUR_USERNAME/nlp_directory")
-# Create your data collection
-nlp_directory$create()
+If the file was an image or some other binary data type, you could instead use the function `.getRaw()` to retrieve the file's content as raw bytes. For more methods on how to get a file using the Data API from a data collection refer to the [API Specification](/developers/api/#get-a-file-or-directory).
 
-# Create the acl object and check if it's the ReadAcl.MY_ALGORITHMS default setting
-acl = nlp_directory$getPermissions()  # Acl object
-acl$read_acl # Returns "MY_ALGORITHMS" Acl Type
+{% if site.enterprise %}
```

Comment:
let's remove the enterprise condition like the earlier comment i made, but making sure we had Enterprise Users header/.info